dashboard2/settings/statistics.md

```diff
@@ -0,0 +1,13 @@
+# Project statistics
+
+![Project statistics screen](../../img/dashboard2/project_statistics.png)
+
+This screen shows you the most important statistics of the project as well as allowing you to add Google Analytics to it.
+
+## Google Analytics
+
+In this section, you can enter a Google Analytics Universal Tracking ID. When you do, the proxy will start reporting the requests it receives to Google Analytics. These are the requests that the proxy processes and thus are billed to you. 
```

Comment:
The data from Universal Analytics is different from the data from Google Analytics, which might confuse users!